doc/reference/configuration/configuration_reference.rst

```diff
@@ -11,6 +11,328 @@ This topic describes all :ref:`configuration parameters <configuration>` provide
 Most of the configuration options described in this reference can be applied to a specific instance, replica set, group, or to all instances globally.
 To do so, you need to define the required option at the :ref:`specified level <configuration_scopes>`.
 
+..  _configuration_reference_audit:
+
+audit_log
+---------
+
+..  admonition:: Enterprise Edition
+    :class: fact
+
+    Configuring ``audit_log`` parameters is available in the `Enterprise Edition <https://www.tarantool.io/compare/>`_ only.
+
+
+The ``audit_log`` section defines configuration parameters related to :ref:`audit logging <enterprise_audit_module>`.
+
+..  NOTE::
+
+    ``audit_log`` can be defined in any :ref:`scope <configuration_scopes>`.
+
+
+* :ref:`audit_log.extract_key <configuration_reference_audit_extract_key>`
+* :ref:`audit_log.file <configuration_reference_audit_file>`
+* :ref:`audit_log.filter <configuration_reference_audit_filter>`
+* :ref:`audit_log.format <configuration_reference_audit_format>`
+* :ref:`audit_log.nonblock <configuration_reference_audit_nonblock>`
+* :ref:`audit_log.pipe <configuration_reference_audit_pipe>`
+* :ref:`audit_log.spaces <configuration_reference_audit_spaces>`
+* :ref:`audit_log.syslog_facility <configuration_reference_audit_syslog-facility>`
+* :ref:`audit_log.syslog_identity <configuration_reference_audit_syslog-identity>`
+* :ref:`audit_log.syslog_server <configuration_reference_audit_syslog-server>`
+* :ref:`audit_log.to <configuration_reference_audit_to>`
+
+..  _configuration_reference_audit_extract_key:
+
+..  confval:: audit_log.extract_key
+
+    **Since:** :doc:`3.0.0 </release/3.0.0>`.
+
+    Specify the logging mode in DML events.
```

Comment:
>  Specify the logging mode in DML events.

I think, this sentence might be dropped as the next one tells exactly what this option does.